examples/camelot_quick_start_notebook.ipynb

```diff
@@ -0,0 +1,413 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/alexfazio/pypdf_table_extraction/blob/main/camelot_quick_start_notebook.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
Is it possbile to have this link to the main repo?

Reply:
I think this might only be possible once it is merged?

Not entirely sure